src/helpers/configuration.sh

```diff
@@ -24,7 +24,7 @@ fi
 if [[ "$(is_git_town_configured)" == false ]]; then
   echo "Git Town hasn't been configured for this repository."
   echo "Please run 'git town config --setup'."
-  echo -n "Would you like to do that now? [y/n] "
+  echo -n "Would you like to do that now?"
```

Comment:
I thought it made more sense to have the `[Y/n]` echoed in the `prompt_yn` function, since the text directly relates to the `case` statement's pattern matching. I don't feel that strongly about this though.